src/codeu/chat/client/commandline/Chat.java

```diff
@@ -108,6 +109,8 @@ public void invoke(Scanner args) {
         System.out.println("  u-sign-in <name>");
         System.out.println("    Sign in as the user with the given name.");
         System.out.println("  exit");
+        System.out.println("  version check");
+        System.out.println("  Display the version of the server.");
```

Comment:
Also put the "version check" command above the "exit" command to keep the menu easy to read